File: C#Integration/PokerProtocol/Commands/Game/PlayerLeftCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using EricUtility;
using PokerProtocol.Commands.Lobby.Response;
using PokerWorld.Game;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Game
{
    public class PlayerLeftCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "gamePLAYER_LEFT";

        private readonly int m_PlayerPos;

        public int PlayerPos
        {
            get { return m_PlayerPos; }
        }

        public PlayerLeftCommand(StringTokenizer argsToken)
        {
            m_PlayerPos = int.Parse(argsToken.NextToken());
        }

        public PlayerLeftCommand(int pos)
        {
            m_PlayerPos = pos;
        }

        public override void Encode(StringBuilder sb)
        {
            Append(sb, m_PlayerPos);
        }
    }
}

File: JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/game/PlayerLeftCommand.java
package bluffinmuffin.protocol.commands.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands.AbstractCommand;

public class PlayerLeftCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return PlayerLeftCommand.COMMAND_NAME;
    }
    
    private final int m_playerPos;
    
    public static String COMMAND_NAME = "gamePLAYER_LEFT";
    
    public PlayerLeftCommand(StringTokenizer argsToken)
    {
        m_playerPos = Integer.parseInt(argsToken.nextToken());
    }
    
    public PlayerLeftCommand(int pos)
    {
        m_playerPos = pos;
    }
    
    @Override
    public void encode(StringBuilder sb)
    {
        append(sb, m_playerPos);
    }
    
    public int getPlayerPos()
    {
        return m_playerPos;
    }
}
